syz-manager/manager.go

```diff
@@ -70,10 +71,15 @@ type Manager struct {
 	maxSignal      map[uint32]struct{}
 	corpusCover    map[uint32]struct{}
 	prios          [][]float32
+	repros         [][]byte
 
 	fuzzers   map[string]*Fuzzer
 	hub       *RpcClient
 	hubCorpus map[hash.Sig]bool
+	hubRepros map[hash.Sig]bool
+
+	reproQueue    []*Crash
```

Comment:
This is now not accesses by multiple goroutines and can be moved back to vmLoop (which also ensures that there are no accidental races on it today or tomorrow).